docs/source/markdown/options/compat-volumes.md

```diff
@@ -0,0 +1,11 @@
+####> This option file is used in:
+####>   podman build, farm build
+####> If file is edited, make sure the changes
+####> are applicable to all of those.
+#### **--compat-volumes**
+
+Handle directories marked using the VOLUME instruction (both in this build, and
+those inherited from base images) such that their contents can only be modified
+by ADD and COPY instructions. Any changes made in those locations by RUN
+instructions will be reverted. Before the introduction of this option, this
+behavior was the default, but it is now disabled by default.
```

Comment:
if the option does not work we cannot documented it

Reply:
...but if the option isn't documented we cannot merge (due to the man page crosscheck). Catch-22. As you said, the option needs to be wired in. (And also like you said, this needs to be done on main first)